src/frontends/tensorflow/src/input_model.cpp

```diff
@@ -115,8 +115,19 @@ void InputModel::InputModelTFImpl::loadPlaces() {
         m_op_places.push_back(op_place);
         m_op_places_map[op_name] = op_place;
         if (op_type == "Placeholder") {
-            auto pshape = node_decoder->get_attribute("shape").as<ov::PartialShape>();
-            auto type = node_decoder->get_attribute("dtype").as<ov::element::Type>();
+            auto pshape = ov::PartialShape::dynamic();
+            auto shape_any = node_decoder->get_attribute("shape");
+            if (shape_any.is<ov::PartialShape>()) {
+                // sometimes shape attribute can be absent in the graph
+                // so we need to check if Any object is initialized first
+                pshape = shape_any.as<ov::PartialShape>();
+            }
```

Comment:
Could you add debug log (OPENVINO_DEBUG) here in case of the shape is absent?

Reply:
Fixed in https://github.com/openvinotoolkit/openvino/pull/14238